app/models/course/requirement/achievement.rb

```diff
@@ -0,0 +1,4 @@
+class Course::Requirement::Achievement < ActiveRecord::Base
+  acts_as :requirement, class_name: 'Course::Requirement'
```

Comment:
`Course::Requirement.name`, so that if we break the identifiers we get a load-time error instead of a runtime error.